### DISCUSSION TITLE
Rest Api

### DISCUSSION THREAD

**Discussion by omer83** [hiddify/Hiddify-Manager#3062] [Content Help کمک به اصلاح محتوا]
Hello,
I developed add and remove user api system for hidedify. If anyone wants other functions, I'll add them.

Add User:
Endpoint: https://www.plumedns.com/projects/hiddify/create_user.php
Post Data: hiddify_url, name, usage_limit, package_days, mode
FORM URL Encoded

Delete User:
Endpoint: https://www.plumedns.com/projects/hiddify/delete_user.php
Post Data: hiddify_url, name, uuid
FORM URL Encoded

hiddify_url = Hiddify admin panel url
name = Username
usage_limit = Usage Limit (GB) 
package_days = Package Days 
mode = daily, weekly, monthly, no_reset

**Comment by sirpasha** [comment]
اینو تیم هیدیفای بررسی کرده؟

**Comment by MehdiSalari** [comment]
متاسفانه کار نمیکنه.
